.gitignore

```diff
@@ -8,6 +8,10 @@ build/
 /.idea/
 #    IntelliJ creates this folder, ignore.
 /dev-tools/missing-doclet/out/
+*.iml
```

Comment:
Hmm... how come you have intellij's old-style files and not the folder-based layout? :)

Reply:
Hmm I seems to recall these files were generated after running `./gradlew idea`, but I removed them in the latest commit anyway.

Reply:
Right... gradlew idea is a plugin - it does generate those files. IntelliJ has native gradle support nowadays. See help/IDEs.txt; perhaps it should be clarified there.